code-of-conduct.md

```diff
@@ -21,28 +20,30 @@ Other languages available:
 - [Ukrainian/Українська](code-of-conduct-languages/uk.md)
 - [Vietnamese/Tiếng Việt](code-of-conduct-languages/vi.md)
 
-### Contributor Code of Conduct
+### Community Code of Conduct
 
-As contributors and maintainers in the CNCF community, and in the interest of fostering
-an open and welcoming community, we pledge to respect all people who contribute
+As contributors, maintainers, and participants in the CNCF community, and in the interest of fostering
+an open and welcoming community, we pledge to respect all people who participate or contribute
 through reporting issues, posting feature requests, updating documentation,
-submitting pull requests or patches, and other activities.
+submitting pull requests or patches, attending conferences or events, or engaging in other community or project activities.
 
-We are committed to making participation in the CNCF community a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression,
-sexual orientation, disability, personal appearance, body size, race, ethnicity, age,
-religion, or nationality.
+We are committed to making participation in the CNCF community a harassment-free experience for everyone, regardless of age, body size, caste, disability, ethnicity, level of experience, family status, gender, gender identity and expression, marital status, military or veteran status, nationality, personal appearance, race, religion, sexual orientation, socieconomic status, tribe, or any other dimension of diversity.
```

Comment:
Copying from the mailing list vote thread "[cncf-toc] [VOTE] Updated Code of Conduct":

> +1B
> 
> Nit: https://github.com/cncf/foundation/pull/575/files#diff-f1a69a41a835b5265e1c58e11395a894ac30fc6b0afe1ca834738955261d2ec5R30 "or any other dimension of diversity." is underdefined; I agree with the (perceived) intention though would prefer more explicit language. It's a nit, not a blocker, though.